source/PyMaterialX/PyMaterialXCore/PyElement.cpp

```diff
@@ -159,6 +160,25 @@ void bindPyElement(py::module& mod)
         .def("_getValue", &mx::ValueElement::getValue)
         .def("_getBoundValue", &mx::ValueElement::getBoundValue)
         .def("_getDefaultValue", &mx::ValueElement::getDefaultValue)
+        .def("setUnit", &mx::ValueElement::setUnit)
+        .def("hasUnit", &mx::ValueElement::hasUnit)
+        .def("getUnit", &mx::ValueElement::getUnit)
+        .def("setUnitType", &mx::ValueElement::setUnitType)
+        .def("hasUnitType", &mx::ValueElement::hasUnitType)
+        .def("getUnitType", &mx::ValueElement::getUnitType)
+        .def_readonly_static("VALUE", &mx::ValueElement::VALUE_ATTRIBUTE)
```

Comment:
To match the conventions of our existing Python bindings, these Python constants should have the "ATTRIBUTE" prefix, and constants like "INTERFACENAME" should have the same underscore as their C++ equivalents (e.g. "INTERFACE_NAME_ATTRIBUTE").